checkov/terraform/checks/resource/ncp/AccessControlGroupRuleDescription.py

```diff
@@ -5,7 +5,7 @@
 class AccessControlGroupRuleDescription(BaseResourceCheck):
     def __init__(self):
         name = "Ensure every access control groups rule has a description"
-        id = "CKV_NCP_002"
+        id = "CKV_NCP_2"
```

Comment:
you adjusted this in the other PR already 🙂

Reply:
Oh Sorry. This adjustment occured when I distribute rules. 
Is there anything else that needs to be fixed in this part?

Reply:
I delete this commit in  PR https://github.com/bridgecrewio/checkov/pull/3624/files.
I'll make this adjust valid on this PR and additional PR about NCP rule 6 to 11 will adjust.

If more issue occur, let me know with comments !